content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -1,391 +1,287 @@
 Activity.Tags:
-    foo: 1
-    bar: Hello, World!
-    baz: [1,2,3]
+    net.host.name: localhost
+    net.host.port: 8080
+    http.method: GET
+    http.scheme: http
+    http.target: /rolldice
+    http.url: http://localhost:8080/rolldice
+    http.flavor: 1.1
+    http.user_agent: curl/8.0.1
+    http.status_code: 200
 Resource associated with Activity:
-    service.name: MyService
-    service.instance.id: 2b07a9ca-29c4-4e01-b0ed-929184b32192
+    service.name: simple-dotnet
+    telemetry.auto.version: 0.7.0
+    telemetry.sdk.name: opentelemetry
+    telemetry.sdk.language: dotnet
+    telemetry.sdk.version: 1.4.0.802
 ```
 
-</details>
-
-You'll notice the `Activity` objects from ASP.NET Core alongside the `Activity`
-we created manually in our controller action.
+At step 6, when stopping the server, you should see an output of all the metrics
+collected (metrics output is shortened for convenience):
 
-### Metrics
+```log
+Export process.runtime.dotnet.gc.collections.count, Number of garbage collections that have occurred since process start., Meter: OpenTelemetry.Instrumentation.Runtime/1.1.0.2
+(2023-08-14T06:12:05.8500776Z, 2023-08-14T06:12:23.7750288Z] generation: gen2 LongSum
+Value: 2
+(2023-08-14T06:12:05.8500776Z, 2023-08-14T06:12:23.7750288Z] generation: gen1 LongSum
+Value: 2
+(2023-08-14T06:12:05.8500776Z, 2023-08-14T06:12:23.7750288Z] generation: gen0 LongSum
+Value: 6
 
-Next we'll add the ASP.NET Core automatically generated metrics to the app.
+...
 
-```csharp
-using OpenTelemetry.Resources;
-using OpenTelemetry.Trace;
-using OpenTelemetry.Metrics;
-
-var builder = WebApplication.CreateBuilder(args);
-
-// .. other setup
-
-builder.Services.AddOpenTelemetry()
-    .WithTracing(/*  .. tracing setup */ )
-    .WithMetrics(metricsProviderBuilder =>
-        metricsProviderBuilder
-            .ConfigureResource(resource => resource
-                .AddService(DiagnosticsConfig.ServiceName))
-            .AddAspNetCoreInstrumentation()
-            .AddConsoleExporter());
-
-// .. other setup
-```
-
-If you run your application now, you'll see a series of metrics output to the
-console. like this.
-
-<details>
-<summary>View example output</summary>
-
-```text
-Export http.server.duration, Measures the duration of inbound HTTP requests., Unit: ms, Meter: OpenTelemetry.Instrumentation.AspNetCore/1.0.0.0
-(2023-02-21T12:38:57.0187781Z, 2023-02-21T12:44:16.9651349Z] http.flavor: 1.1 http.method: GET http.route: {controller=Home}/{action=Index}/{id?} http.scheme: http http.status_code: 200 net.host.name: localhost net.host.port: 5123 Histogram
-Value: Sum: 373.4504 Count: 1 Min: 373.4504 Max: 373.4504
+Export http.client.duration, Measures the duration of outbound HTTP requests., Unit: ms, Meter: OpenTelemetry.Instrumentation.Http/1.0.0.0
+(2023-08-14T06:12:06.2661140Z, 2023-08-14T06:12:23.7750388Z] http.flavor: 1.1 http.method: POST http.scheme: https http.status_code: 200 net.peer.name: dc.services.visualstudio.com Histogram
```

Comment:
interesting metric! it looks like the app is connecting to application insights (dc.services url), and that got collected as http client metric.

Reply:
the app itself is not making any http-client calls, so this could be confusing to see in the output!

Reply:
Yeah agreed. I'm probably okay with leaving it, but ideally this wouldn't connect to any external service so that people who just run locally can easily compare outputs to their own.

Reply:
This looks like telemetry from dotnet CLI, there is a `dotnet run` step. It may go away if `DOTNET_CLI_TELEMETRY_OPTOUT` is set to `true`.